tools/run_tests/run_interop_tests.py

```diff
@@ -589,6 +607,12 @@ def aggregate_http2_results(stdout):
                   action='store_const',
                   const=True,
                   help='Run cloud_to_prod_auth tests.')
+argp.add_argument('-p', '--prod_servers',
```

Comment:
nit:  short form of the argument "-p" is not very understandable. Can we get rid of the short form and only use --prod_server?